var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -10,16 +10,12 @@ class PyCython(PythonPackage):
     """The Cython compiler for writing C extensions for the Python language."""
 
     homepage = "https://github.com/cython/cython"
-    pypi = "cython/Cython-0.29.21.tar.gz"
+    pypi = "cython/cython-3.0.11.tar.gz"
     tags = ["build-tools"]
 
     license("Apache-2.0")
 
-    version(
-        "3.0.11",
-        sha256="7146dd2af8682b4ca61331851e6aebce9fe5158e75300343f80c07ca80b1faff",
-        url="https://files.pythonhosted.org/packages/source/cython/cython-3.0.11.tar.gz",
```

Comment:
This URL isn't actually correct, it's missing a `/c/` in the path. Not sure how this ever worked.